lib/src/acter_avatar.dart

```diff
@@ -12,48 +12,23 @@ final log = Logger('ActerAvatar');
 /// image resizing (for memory purposes) has to be done beforehand, the image
 /// is shown as given.
 class ActerAvatar extends StatefulWidget {
-  /// Set the display mode for this Avatar.
-  final DisplayMode mode;
```

Comment:
Firstly, the most major change in this PR is to wrap all these into one configurable parameter `AvatarOptions` which  can be initialised in different ways depending upon type of avatar. I found out that this approach was reducing a lot of code lines when played in actual widget tree.